src/css/css-document.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {Node, NodeVisitor} from 'shady-css-parser';
+
+import {Document} from '../parser/document';
+
+export class CssDocument extends Document<Node, NodeVisitor> {
+  type = 'css';
```

Comment:
This results in a constructor being generated that uses `...args` as a parameter, which isn't supported in node v4.